### DISCUSSION TITLE
Custom CSS does not work for me

### DISCUSSION THREAD

**Discussion by Opicka879** [dani-garcia/vaultwarden#5280] [Q&A] [answered]
Hi,

i did try to set my CSS to overwrite logos, but nothing happens. What am i doing wrong? Version 1.32.6.

My user.vaultwarden.scss.hbs:

```
bit-icon > svg {
  display: none !important;
}
bit-icon::before {
  display: block;
  content: "" !important;
  width: 175px !important;
  height: 30px !important;
  background-repeat: no-repeat !important;
  background-size: contain;
}
app-user-layout bit-icon::before {
  background-image: url(../images/logo-my-vault-web.svg) !important;
}
app-organization-layout bit-icon::before {
  background-image: url(../images/logo-my-vault-web-admin.svg) !important;
}
```

**Comment by stefan0xC** [comment, marked as answer]
The CSS is not yet included in the web-vault. So you'd need to either patch the web-vault yourself or wait for https://github.com/dani-garcia/bw_web_builds/pull/180 to be not only merged but released as well.

**Reply by Opicka879** [reply]
> The CSS is not yet included in the web-vault. So you'd need to either patch the web-vault yourself or wait for [dani-garcia/bw_web_builds#180](https://github.com/dani-garcia/bw_web_builds/pull/180) to be not only merged but released as well.

Ah, ok then. I was confused by 1.32.5. release notes. There is a line "Add dynamic CSS support by @BlackDex in #4940", so I thought it is implemented already.

**Reply by BlackDex** [reply]
Yes, it is in there, though the web-vault does not have support for it yet.

I just finished testing the recently released v2024.12.0 web-vault, and made changes where needed for it.
I think it works ok.
